Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.36271&attempt_id=023f96e0-caa0-497a-84e6-9e24d399d084)

<details><summary>Partial log (click to expand)</summary><p>

```
        QDBusConnection sessionBus = QDBusConnection::sessionBus();
                                     ^
src/main.cpp:251:35: error: use of undeclared identifier 'sessionBus'
        utils.checkDBusConnection(sessionBus);
                                  ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
5 warnings and 20 errors generated.
make: *** [Makefile:1378: main.o] Error 1
builder for '/nix/store/5lgnbki85k8rpncp95qbiz44b3fas7qa-flameshot-0.5.1.drv' failed with exit code 2
error: build of '/nix/store/5lgnbki85k8rpncp95qbiz44b3fas7qa-flameshot-0.5.1.drv' failed
```
</p></details>

--- a/pkgs/tools/misc/flameshot/default.nix
+++ b/pkgs/tools/misc/flameshot/default.nix
@@ -2,7 +2,7 @@
 
 stdenv.mkDerivation rec {
   name = "flameshot-${version}";
-  version = "0.5.0";
+  version = "0.5.1";
 
   nativeBuildInputs = [ qmake qttools ];
   buildInputs = [ qtbase ];
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
     owner = "lupoDharkael";
     repo = "flameshot";
     rev = "v${version}";
-    sha256 = "1fy4il7rdj294l9cs642hx23bry25j9phn37274r2b87hwzy1rrv";
+    sha256 = "13h77np93r796jf289v4r687cmnpqkyqs34dm9gif4akaig74ky0";
   };
 
   enableParallelBuilding = true;
@@ -34,6 +34,6 @@ stdenv.mkDerivation rec {
     homepage = https://github.com/lupoDharkael/flameshot;
     maintainers = [ maintainers.scode ];
     license = stdenv.lib.licenses.gpl3;
-    platforms = stdenv.lib.platforms.all;
+    platforms = stdenv.lib.platforms.linux;
   };
 }

